docs/current/guides/snippets/use-services/expose-host-services-to-container/main.py

```diff
@@ -9,7 +9,7 @@ async def main():
     # create Dagger client
     async with dagger.Connection(dagger.Config(log_output=sys.stderr)) as client:
         # expose host service on port 3306
-        host_srv = client.host().service(frontend=3306, backend=3306)
+        host_srv = client.host().service([dagger.PortForward(frontend=3306, backend=3306, protocol='TCP')])
```

Comment:
Should use the enum for protocol and (nit) prefer to follow the defined [parameter order](https://dagger-io.readthedocs.io/en/latest/client.html#dagger.PortForward):
```suggestion
        host_srv = client.host().service([dagger.PortForward(backend=3306, frontend=3306, protocol=dagger.NetworkProtocol.TCP)])
```